Dockerfile

```diff
@@ -0,0 +1,39 @@
+# FROM ubuntu-upstart:precise
+# http://phusion.github.io/baseimage-docker/
+FROM phusion/baseimage:0.9.9
+
+# Install pre-pre-requisites.
+RUN apt-get update
```

Comment:
For peeps playing along at home I'm incorrect with that comment! @rmccue mentioned this in our Slack.

>Docker can handle daemonisation fine
It's just systemd that doesn't like not having access to the root filesystem